docs/tutorial.ipynb

```diff
@@ -486,7 +485,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.6"
+   "version": "3.11.2"
```

Comment:
Same question, is this related to your change?

Reply:
Just to the version of python running on my machine...